app/graphql/types/library_entry.rb

```diff
@@ -15,9 +15,9 @@ class Types::LibraryEntry < Types::BaseObject
     null: false,
     description: ''
 
-  # field :media_reaction, Types::MediaReaction,
-  #   null: true,
-  #   description: 'Reaction for this specific media'
+  field :media_reaction, Types::MediaReaction,
```

Comment:
Thinking maybe this should just be `:reaction` to cut unnecessary context

Reply:
yes.